src/subclass/interface.rs

```diff
@@ -249,14 +249,6 @@ pub trait ObjectInterfaceExt: ObjectInterface {
 
 impl<T: ObjectInterface> ObjectInterfaceExt for T {}
 
-unsafe extern "C" fn interface_init<T: ObjectInterface>(
```

Comment:
This not. While it is close to a trampoline in principle, it's even closer to an actual function implementation. I'd keep this outside.